pkg/test/e2e/kafka/kafka_end_to_end_test.go

```diff
@@ -342,17 +342,19 @@ func runCommandGetOutput(t *testing.T, command string) string {
 		assert.Fail(t, msg)
 	}
 	output := outBuf.Bytes()
-	// strip the line feed from the end of the output string and the quotation marks
-	output = output[1 : len(output)-2]
+	// strip the line feed from the end of the output string
+	output = output[0 : len(output)-1]
```

Comment:
Better to not strip anything and let each individual program do what it needs with the output.